strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ArtifactSearchServiceImpl.java

```diff
@@ -18,8 +15,7 @@
         implements ArtifactSearchService
 {
 
-    @Inject
-    private OrientDbSearchProvider orientDbSearchProvider;
+    private SearchProvider orientDbSearchProvider;
```

Comment:
Is this some leftover? This should probably be called something else. Could you please have another look? Thanks! :)

Reply:
renamed to `searchProvider`, but anyway all that related to the search should be re-implemented as we plan to do in the near future